examples/stack/connection.html

```diff
@@ -0,0 +1,57 @@
+<html>
+<head>
+  <link rel="stylesheet" type="text/css" href="cozy-ui.min.css" />
+  <link rel="stylesheet" type="text/css" href="cozy-ui.utils.min.css" />
+  <style>
+    {{ STYLE }}
+  </style>
+</head>
+<body>
+<div role="application">
+  <main class="wizard">
+    <form method="POST" action="/sharings/32176e6db46e8106bacf29f53b19abb4/discovery" class="wizard-wrapper">
+      <div role="region" class="wizard-main">
+        <input type="hidden" name="state" value="" />
+        <input type="hidden" name="sharecode" value="eyJhbGciOiJIUzUxMiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJzaGFyZSIsImlhdCI6MTYwMTI4MDI5MiwiaXNzIjoicGJyb3duZS5teWNvenkuY2xvdWQiLCJzdWIiOiJwdC5icm93bmUrdGVzdDFAZ21haWwuY29tIn0.-wZ6vEAY0p-LthjXlXtDWSUfjxjbzt8zyQbvgrrjRmtE2MAO_N85GX14DSpiU1eiusi54mv7TVXIEXl5ppju1Q" />
+        <div class="c-avatar u-mh-auto u-mv-0-m u-mv-1 c-avatar--xlarge">
+          <img class="c-avatar-image" src="/public/avatar" alt="" />
```

Comment:
We don't an image that we can use here? I think it would be better to have any image that a 404

Reply:
I added an image, thanks.